### PR TITLE
Updated colors in new light and dark files

### PR DESCRIPTION
**Pull request by Ijin08**
Updated colors in light, dark and theme files.
In template file basic colors uses variables from dark/light files. 
Also changed $green to $green-basic in some files

### CODE REVIEW

File: packages/grafana-ui/src/themes/_variables.light.scss.tmpl.ts [hunk 1]
```diff
@@ -182,11 +180,11 @@ $btn-drag-image: '../img/grab_light.svg';
 $input-bg: $white;
 $input-bg-disabled: $gray-5;
 
-$input-color: $dark-5;
+$input-color: $dark-2;
```
Comment (torkelo) [inline packages/grafana-ui/src/themes/_variables.light.scss.tmpl.ts]: seems like big change?

File: packages/grafana-ui/src/themes/_variables.light.scss.tmpl.ts [hunk 2]
```diff
@@ -72,10 +70,10 @@ $page-bg: $gray-7;
 
 $body-color: $gray-1;
 $text-color: $gray-1;
-$text-color-strong: $dark-2;
+$text-color-strong: $dark-1;
 $text-color-weak: $gray-2;
 $text-color-faint: $gray-4;
-$text-color-emphasis: $dark-5;
+$text-color-emphasis: $dark-2;
```
Comment (torkelo) [inline packages/grafana-ui/src/themes/_variables.light.scss.tmpl.ts]: big change?

File: packages/grafana-ui/src/themes/_variables.light.scss.tmpl.ts (hunk 1 above)
Comment (Ijin08) [reply packages/grafana-ui/src/themes/_variables.light.scss.tmpl.ts]: Actually nothing has changed from master. Before I removed some dark variables. So i renamed dark-5 to dark-2.

**Comment by torkelo** [comment]
merging this but there is still a lot of missmatch between code & sass file. 

All high level color names need to be taken from theme code object in template as well, not just the basic colors 

https://github.com/grafana/grafana/blob/master/packages/grafana-ui/src/themes/dark.ts#L60

**Pull request merged**